webtool/test_api.py

```
@@ -1516,7 +1521,10 @@ def test_projekt_umbenennen_bleibt_grob_gesperrt(client, monkeypatch):
     monkeypatch.setattr(jobs_mod, "betrifft", lambda name, base, **kw: None)   # keine Datei betroffen
     monkeypatch.setattr(jobs_mod, "active_for", lambda name: [{"id": "j1", "kind": "transcribe"}])
     r = client.post("/api/projects/Demo/rename", json={"name": "Neu"})
-    assert r.status_code == 409 and "Transkription" in r.json()["detail"]
+    # „Verarbeitung" seit #442 — der transcribe-Job korrigiert mit, „Transkription" war
+    # waehrend seiner Korrekturphase falsch. Der Test misst die SPERRE, das Wort steht hier
+    # nur mit; es bleibt trotzdem eine Zusicherung und wird deshalb nachgezogen statt gelockert.
+    assert r.status_code == 409 and "Verarbeitung" in r.json()["detail"]
```

Ruff (0.16.3)

[warning] Assertion should be broken down into multiple parts

Break down assertion into multiple parts

(PT018)